src/settings/index.tsx

```diff
@@ -259,6 +300,19 @@ ${this.props.lastSevereError}`;
     });
   }
 
+  private setTorrentTrackers<K extends keyof TorrentTrackerSettings>(
+    key: K,
+    value: TorrentTrackerSettings[K],
+  ) {
+    console.log(key, value);
```

Comment:
Could you go through and remove some of these console logs? It's okay to leave the ones that provide useful debugging information, but there's a couple like this one that don't.

Reply:
Sure. I'll check all codes again.